topic7/src/main/java/models/Course.java

```diff
@@ -0,0 +1,86 @@
+package models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "course")

```

Comment:
Please validate if the name of the table is "course" in lowercase and if the fields are being correctly mapped (it that's not the case you could use @Column(name = "<fied name>"), when <field name> could be, for example, course_name